cfme/networks/topology.py

```diff
@@ -0,0 +1,200 @@
+import re
+import time
+
+from navmazing import NavigateToAttribute
+
+from cfme.networks.topology_view import TopologyView
+from cfme.utils.appliance import Navigatable
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from wait_for import wait_for
+
+
+class Topology(Navigatable):
```

Comment:
**REQUIRED**
It should be a collection and inherits `BaseCollection`.

Reply:
Should be the collection class used even if exists only 1 always active object like network topology?

Reply:
I think yes, but we can also ask @mfalesni and @psav.